### PR TITLE
Add uncomplete commands for testing

### PR DESCRIPTION
**Pull request by PabloPintor**
Adds commands to uncomplete bingo challenges for testing purposes
`/bingo uncomplete [player]` to set all challenges to uncomplete
`/bingo uncomplete [player] [index]`  to set an specific challenge to uncomplete

### CODE REVIEW

File: src/main/java/tc/oc/bingo/commands/CardCommand.java [hunk 1]
```diff
@@ -87,6 +89,36 @@ public void bingoCardComplete(CommandSender sender, OnlinePlayer target, int ind
...
+    if (!(Bingo.get().getBingoDatabase() instanceof MockDatabase)) {
+      sender.sendMessage("This command is only available when using a mocked database");
+      return;
+    }
+
+    Bingo.get()
+        .getPlayerCard(target.getPlayer().getUniqueId())
+        .whenComplete(
+            (c, t) -> {
+              if (Objects.isNull(c)) {
```
Comment (Pugzy) [inline src/main/java/tc/oc/bingo/commands/CardCommand.java]: Can you fix the Objects.isNull bits for actual == null etc

File: src/main/java/tc/oc/bingo/commands/CardCommand.java [hunk 2]
```diff
@@ -87,6 +89,36 @@ public void bingoCardComplete(CommandSender sender, OnlinePlayer target, int ind
...
+      CommandSender sender, OnlinePlayer target, @Optional Integer index) {
+    if (!(Bingo.get().getBingoDatabase() instanceof MockDatabase)) {
+      sender.sendMessage("This command is only available when using a mocked database");
+      return;
+    }
+
+    Bingo.get()
+        .getPlayerCard(target.getPlayer().getUniqueId())
+        .whenComplete(
+            (c, t) -> {
```
Comment (Pugzy) [inline src/main/java/tc/oc/bingo/commands/CardCommand.java]: Also give names to the (c, t)

**Review by Pugzy — COMMENTED** [review]
lbtm

File: src/main/java/tc/oc/bingo/commands/CardCommand.java (hunk 2 above)
Comment (PabloPintor) [reply src/main/java/tc/oc/bingo/commands/CardCommand.java]: Renamed them to "(card, throwable)"

File: src/main/java/tc/oc/bingo/commands/CardCommand.java (hunk 1 above)
Comment (PabloPintor) [reply src/main/java/tc/oc/bingo/commands/CardCommand.java]: This is a valid way of checking if an object is null

Comment (Pugzy) [reply src/main/java/tc/oc/bingo/commands/CardCommand.java]: It is but then you do it another way on line https://github.com/OvercastCommunity/Bingo/pull/7/files#diff-64654d382165b413ecab53ae2416bf6071a85df756e650474ee5323086972200R113 please stick with the == null != null format for consistency across the project.

Comment (PabloPintor) [reply src/main/java/tc/oc/bingo/commands/CardCommand.java]: Changed format to object == null

**Review by Pablete1234 — APPROVED** [review]

**Pull request merged**